disperser/apiserver/disperse_blob_v2.go

```diff
@@ -23,6 +23,7 @@ func (s *DispersalServerV2) DisperseBlob(ctx context.Context, req *pb.DisperseBl
 		s.metrics.reportDisperseBlobLatency(time.Since(start))
 	}()
 
+	s.logger.Debug("received a new blob dispersal request", "payment", req.GetBlobHeader().GetPaymentHeader())
```

Comment:
Is there a reason you added a new message for this, instead of adding these fields to the log in line 50? Log text is the same for both, with just different key value pairs